administrator/components/com_joomlaupdate/tmpl/upload/default.php

```diff
@@ -83,8 +83,9 @@
 		</div>
 	</div>
 
-	<div class="form-check mb-3">
-		<input class="form-check-input me-2" type="checkbox" value="" id="joomlaupdate-confirm-backup">
+	<div class="form-check mb-3 <?php echo $this->noBackupCheck ? 'd-none' : '' ?>">
```

Comment:
```suggestion
	<div class="form-check mb-3<?php echo $this->noBackupCheck ? ' d-none' : '' ?>">
```

minor thing but this way you dont have an extra space at the end when there is a backup check

Reply:
This is intentional. An extra space is far less important than someone cocking this up 3–5 years down the line because they missed the space before the class name.

Reply:
if they miss the space then they will see it doesn't work.

Reply:
My experience is that they will probably miss it because it's an optional feature and class changes are very likely to be done in bulk.

Reply:
ymmv